ext/lib/mgmt/mcumgr/cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```diff
@@ -89,16 +111,16 @@ zephyr_img_mgmt_flash_area_id(int slot)
 
     switch (slot) {
     case 0:
-        fa_id = DT_FLASH_AREA_IMAGE_0_ID;
+        fa_id = FLASH_AREA_IMAGE_PRIMARY;
         break;
 
     case 1:
-        fa_id = DT_FLASH_AREA_IMAGE_1_ID;
+        fa_id = FLASH_AREA_IMAGE_SECONDARY;
         break;
 
     default:
         assert(0);
-        fa_id = DT_FLASH_AREA_IMAGE_1_ID;
+        fa_id = FLASH_AREA_IMAGE_PRIMARY;
```

Comment:
Shouldn't this be `SECONDARY` ?

Reply:
that's correct will fix it

Reply:
I don't know why you would default to the secondary after an assert but I assume it checks the primary first